client/battle/CBattleInterface.cpp

```diff
@@ -3083,12 +3099,19 @@ void CBattleInterface::showBackgroundImage(SDL_Surface *to)
 	}
 }
 
-void CBattleInterface::showAbsoluteObstacles(SDL_Surface *to)
+void CBattleInterface::showAbsoluteObstacles(SDL_Surface * to)

```

Comment:
such a note only, these rendering things that render obstacles will make conflicts with my branch but nothing can be do with it

Reply:
fixed